EapImport/src/main/java/de/hpi/unicorn/importer/json/JsonExporter.java

```diff
@@ -88,4 +89,53 @@ public static File generateExportFileWithDependencies(final EapEventType eventTy
         }
         return file;
     }
+
+    /**

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=checkstyle%3AJavadoc_Style_BP2016)